2016-01-15.md

```diff
@@ -0,0 +1,9 @@
+# This week's most excellent reads
+
+## Elixir
+
+* [Pattern Matching Complex Strings](http://thepugautomatic.com/2016/01/pattern-matching-complex-strings)
```

Comment:
@duggiefresh Does this article highlight a change in how pattern matching is done in Elixir? If not, I would prefer to just link to it in the Slack engineering channel. As we discussed with Lauren earlier this week, we're going to try to steer clear of tutorials and make the Canon more about new features, stuff that we would enforce in PRs etc. 

Agree? Disagree?